data_adapter_oemof/calculations.py

```diff
@@ -214,10 +214,16 @@ def handle_nans(group_df: pd.DataFrame) -> pd.DataFrame:
         if column in ["method", "source", "comment", "bandwidth_type"]:
             continue
 
-        if group_df[column].nunique(dropna=False) > 1:
+        if group_df.isna().sum>0:
```

Comment:
I would remove this check completely as I think this costs more than simply fillna all columns which need to be checked...